<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="SYSR159" failures="1" tests="14" name="SeleniumJavaFramework.ReachHealthTests" time="214.516" errors="0" timestamp="2022-02-12T16:00:06 IST" skipped="0">
  <testcase classname="SeleniumJavaFramework.ReachHealthTests" name="TC09_verifyResourcesPageSections" time="16.679"/>
  <system-out/>
  <testcase classname="SeleniumJavaFramework.ReachHealthTests" name="TC07_verifyGoalsPageSections" time="12.463"/>
  <system-out/>
  <testcase classname="SeleniumJavaFramework.ReachHealthTests" name="TC08_verifyActivityPageSections" time="11.541"/>
  <system-out/>
  <testcase classname="SeleniumJavaFramework.ReachHealthTests" name="TC04_verifyDashboardPageSections" time="16.540"/>
  <system-out/>
  <testcase classname="SeleniumJavaFramework.ReachHealthTests" name="TC11_validateUserAccountSettings" time="12.653"/>
  <system-out/>
  <testcase classname="SeleniumJavaFramework.ReachHealthTests" name="TC14_UpdateQuealthAssessmentWithGoodInputs" time="27.708"/>
  <system-out/>
  <testcase classname="SeleniumJavaFramework.ReachHealthTests" name="TC01_loginToReachHealthWithValidCredentials" time="13.216"/>
  <system-out/>
  <testcase classname="SeleniumJavaFramework.ReachHealthTests" name="TC03_validateAllPageTabs" time="17.004">
    <failure message="expected [https://demo.reachhealth.io/#/goals/create] but found [https://demo.reachhealth.io/#/goals]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [https://demo.reachhealth.io/#/goals/create] but found [https://demo.reachhealth.io/#/goals]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at SeleniumJavaFramework.ReachHealthTests.TC03_validateAllPageTabs(ReachHealthTests.java:212)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- TC03_validateAllPageTabs -->
  <system-out/>
  <testcase classname="SeleniumJavaFramework.ReachHealthTests" name="TC02_loginToReachHealthWithInvalidCredentials" time="6.297"/>
  <system-out/>
  <testcase classname="SeleniumJavaFramework.ReachHealthTests" name="TC12_validateUserAchievements" time="12.442"/>
  <system-out/>
  <testcase classname="SeleniumJavaFramework.ReachHealthTests" name="TC05_viewQuealthReport" time="12.132"/>
  <system-out/>
  <testcase classname="SeleniumJavaFramework.ReachHealthTests" name="TC13_UpdateQuealthAssessmentWithPoorInputs" time="22.254"/>
  <system-out/>
  <testcase classname="SeleniumJavaFramework.ReachHealthTests" name="TC10_createAGoal" time="18.406"/>
  <system-out/>
  <testcase classname="SeleniumJavaFramework.ReachHealthTests" name="TC06_verifyAssessmentPageSections" time="15.181"/>
  <system-out/>
</testsuite> <!-- SeleniumJavaFramework.ReachHealthTests -->
